src/components/Dialogs/RegionDialog/EllipticalRegionForm/EllipticalRegionForm.tsx

```diff
@@ -256,26 +256,30 @@ export class EllipticalRegionForm extends React.Component<{ region: RegionStore,
             sizeHeightInput = <NumericInput selectAllOnFocus={true} buttonPosition="none" placeholder="Semi-minor" value={size.y} onBlur={this.handleMinorAxisChange} onKeyDown={this.handleMinorAxisChange}/>;
         } else {
             sizeWidthInput = (
-                <NumericInput
-                    allowNumericCharactersOnly={false}
-                    buttonPosition="none"
-                    placeholder="Semi-major"
-                    disabled={!this.props.wcsInfo}
-                    value={this.sizeWCS ? this.sizeWCS.x : ""}
-                    onBlur={this.handleMajorAxisWCSChange}
-                    onKeyDown={this.handleMajorAxisWCSChange}
-                />
+                <Tooltip content={"Format: arcsec(\"), arcmin('), or degrees(deg)"} position={Position.BOTTOM}>
+                    <NumericInput
+                        allowNumericCharactersOnly={false}
+                        buttonPosition="none"
+                        placeholder="Semi-major"
+                        disabled={!this.props.wcsInfo}
+                        value={this.sizeWCS ? this.sizeWCS.x : ""}
+                        onBlur={this.handleMajorAxisWCSChange}
+                        onKeyDown={this.handleMajorAxisWCSChange}
+                    />
+                </Tooltip>
             );
             sizeHeightInput = (
-                <NumericInput
-                    allowNumericCharactersOnly={false}
-                    buttonPosition="none"
-                    placeholder="Semi-minor"
-                    disabled={!this.props.wcsInfo}
-                    value={this.sizeWCS ? this.sizeWCS.y : ""}
-                    onBlur={this.handleMinorAxisWCSChange}
-                    onKeyDown={this.handleMinorAxisWCSChange}
-                />
+                <Tooltip content={"Format: arcsec(\"), arcmin('), or degrees(deg)"} position={Position.BOTTOM}>
```

Comment:
Can we increase the `hoverOpenDelay` for these tooltips? Otherwise they seem a bit invasive